src/components/privateRoute/index.test.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { expect } from 'chai';
+import { mount } from 'enzyme';
+import { MemoryRouter, Route } from 'react-router';
+import { PrivateRouteRender } from './index';
+
+const Public = () => <h1>Public</h1>;
+const Private = () => <h1>Private</h1>;
+
+describe('<PrivateRouteRender />', () => {
```

Comment:
Please use just `PrivateRouteRender` to make it consistent with other Components' `describe`s